src/selectors/roughHeights.js

```diff
@@ -120,19 +120,16 @@ function roughHeights(
    */
   let minAvailableHeight = roughAvailableHeight;
   let maxAvailableHeight = roughAvailableHeight;
-  let minAvailableWidth = roughAvailableWidth;
+  const minAvailableWidth = roughAvailableWidth - scrollbarYWidth;
```

Comment:
`minAvailableWidth` is used only in selector/scrollbarVisisble, and it will be the availableWidth when vertical scrollbar is visible.

Reply:
This is going to subtract the width of the vertical scrollbar regardless of whether the vertical scrollbar is visible or not.
I think you want to keep this below `case ScrollbarState.JOINT_SCROLLBARS:` ?

Reply:
The only place where it used is this https://github.com/schrodinger/fixed-data-table-2/blob/column-virtualization/src/selectors/scrollbarsVisible.js#L66. (i.e for setting availableWidth when vertical scrollbar present). and in that case we want to subtract scrollbar width.